.azure-pipelines/Linux-CI.yml

```diff
@@ -134,18 +125,4 @@ jobs:
         exit 1
       fi
 
-      # Mypy only works with Python 3
-      if [ "$(python.version)" != "2.7" ]; then
```

Comment:
why remove this? The check is for !2.7

Reply:
ah thank you for pointing that out. misread the code, will update!

Reply:
well, actually since python version will never equal to 2.7, is the if statement still needed here?

Reply:
I will put back the logic, but no longer wrap it in if statement :)

Reply:
Updated! PTAL when you can, thanks again for the review!